src/components/map/Map.tsx

```diff
@@ -149,6 +160,7 @@ class LeafletMap extends React.Component<IMapProps, IMapState> {
                     />
                     <NodeLayer
                         nodes={visibleNodes}
+                        firstNodes={this.getFirstNodes()}
```

Comment:
Would a more descriptive name maybe be: startLocations?
Previously, we also only had Nodes in NodeLayer, but with this change we also have startLocations so NodeLayer name is no longer so descriptive. We could maybe change this to MarkerLayer, or something similar. Open for suggestions.